mupl/file_validator.py

```diff
@@ -244,6 +246,204 @@ def process_zip_name(self) -> "bool":
         self.publish_date = self._get_publish_date()
         return True
 
+    def process_zip_name_extanded(self) -> "bool":
```

Comment:
This function makes very little sense as part of the class.  There is code in-between internal methods, one of them `match_file` redeclaring half of the class' properties, copypaste of existing code, and overall it's a second weird implementation of process_zip_name that is way harder to read, does not make much use of the regex (because you can't) and that entirely breaks the second the tree format changes.
Also oneshot chapter number is 000 not 0000.